index.js

```diff
@@ -440,6 +444,16 @@ module.exports = {
     return checker.exists();
   },
 
+  _emberJqueryDependencyPresent() {
+    if (this.project.name && this.project.name() === '@ember/jquery') {
+      return true;
+    }
+
+    let checker = new VersionChecker(this.parent).for('@ember/jquery', 'npm');
```

Comment:
> Should we require that @ember/jquery is a project dependency only (it seems somewhat odd for an addon to depend on it)?

Really, why? 
In the RFC it was mentioned that addons that *require* jquery should add it to their dependencies (https://github.com/emberjs/rfcs/blob/master/text/0386-remove-jquery.md#extend-emberjquery-package). 

What would be an alternative, if the addon requires jQuery? (thinking of a jquery plugin wrapper addon) Checking the optional feature flag and the existence of `@ember/jquery` and intentionally throwing an exception during the build when jquery is not present? (as we have no real peer dependencies)

Reply:
> Really, why?

Because we `app.import` it every time its included:

https://github.com/emberjs/ember-jquery/blob/81b3d50ca278e83e11a5bac1e7eb16064b75e025/index.js#L11

Now, that seems fine (probably) because we deduplicate `app.import`'s for the same path...

Reply:
Sorry, I just went in a semi-circle there. Carry on. 😸